torch/_torch_docs.py

```diff
@@ -143,57 +143,44 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.add,
            r"""
-.. function:: add(input, other, out=None)
+.. function:: add(input, other, alpha=1, out=None)
```

Comment:
Why did you combine these functions?  I don't think it's good in the documentation to start with "here's the most advanced form that you will never use".

Reply:
I had removed it to be consistent with the related PR which also removed it, but now re-included both variants in this PR.